projects/admin/src/app/circulation/checkin/checkin.component.ts

```diff
@@ -157,14 +157,19 @@ export class CheckinComponent implements OnInit {
       this.isLoading = true;
       this._patronService
         .getPatron(barcode)
-        .pipe(map(patron => (patron.displayPatronMode = false)))
+        .pipe(map(patron => {
+          if (patron) {
+            patron.displayPatronMode = false;
+          }
+        }))
         .subscribe(
-          () => (this.isLoading = false),
-          error =>
+          () => this.isLoading = false,
+          error => {
             this._toastService.error(
               error.message,
               this._translate.instant('Checkin')
-            ),
+            );
+          },
```

Comment:
I think you have to keep the curly braces in each functions, because without them, the function will return the result of the single instruction and that's not we want.

Reply:
Done